esileapclient/tests/functional/README.md

```diff
@@ -0,0 +1,50 @@
+# esi-leap Client Functional Tests
+
+These tests are designed to ensure the esi-leap services and the esi-leap CLI client function as expected when interacting with each other.
+
+### Prerequisites
+
+These tests are intended to be ran against a functioning OpenStack cloud with esi-leap services enabled and running (https://github.com/CCI-MOC/esi-leap). It assumes that the esi-leap.conf file is located at `/etc/esi-leap/esi-leap.conf`; however if this is not the case, the location of the file can be specified by setting the environment variable `OS_ESI_CFG_PATH` equal to the file's location. This can also be used to provide a custom config file for use with these tests if desired. Additionally, since these tests make use of dummy nodes, the configuration file requires a value for the `dummy_node_dir` option in the `[dummy_node]` section of esi-leap.conf.
```

Comment:
"to be run"

Reply:
"equal to the file's location" feels extraneous

Reply:
Good catches!